library/src/webapp/js/sakai-reminder.js

```diff
@@ -0,0 +1,25 @@
+(function() {
+    this.SakaiReminder = function() {
+    }
+
+    SakaiReminder.prototype = {
+        init: function() {
+        },
+
+        getAll: function() {
+            return JSON.parse(localStorage.getItem(portal.user.id + '-sakaiReminder'));
+        },
+
+        new: function(text) {
+            var predictions = JSON.parse(localStorage.getItem(portal.user.id + '-sakaiReminder'));
```

Comment:
What does portal.user.id map to?

Reply:
The user internal id, what we agreed on the core call meeting.

Reply:
Thanks for confirming, @mpellicer!